Sources/Commands/init.swift

```diff
@@ -17,6 +17,16 @@ import func Utility.fputs
 import func Utility.makeDirectories
 import struct Utility.Path
 
+private extension FSProxy {
+    /// Write to a file from a stream producer.
+    mutating func writeFileContents(_ path: String, body: @noescape (OutputByteStream) -> ()) throws {
```

Comment:
Yup! That is why I factored it out into its own function, but I'm not sure where to put it yet (not sure it belongs as a general purpose extension on FSProxy) so I figured I would leave it here to start.

Where do you think it should go?